components/02-elements/swatch/_swatch.scss

```diff
@@ -0,0 +1,86 @@
+$swatch-font-size        : $font-size-small !default;
+$swatch-margin           : 15px !default;
+$swatch-width            : 48px !default;
+$swatch-transition       : $transition-base !default;
+
+$swatch-background       : $white !default;
+$swatch-color            : $gray-light !default;
+$swatch-border-color     : $swatch-color !default;
+
+$swatch-color-hover      : $color-primary !default;
+
+$swatch-background-active: $color-primary !default;
+$swatch-border-active    : $color-primary !default;
```

Comment:
It's not whole border, but just a color.

Reply:
corrected